### ISSUE TITLE
Molecular Editing: How to get result with mean and standard deviation in supplementary Table 13?

### ISSUE THREAD

**Issue by spoilvoid** [chao1224/MoleculeSTM#36]
Your results in [supplementary Table 13](https://github.com/user-attachments/assets/8aadd44a-affa-4649-920f-ab0b57c44872) are the last two decimal places of the percentages, and the accuracy of the test input is based on only 200 SMILES.  Thus, You must have conducted multiple experiments. Could you let me know how you performed these multiple experiments?
You just change the adapter learning rate or change other configurations like editing epochs and editing lr? 
I hold this question because you say 2 key hyperparameters are the learning rate and $\lambda$ in [supplementary Information](https://github.com/user-attachments/assets/346aaf71-5359-4b83-b67d-07e6ef686c1a). But in your [code](https://github.com/chao1224/MoleculeSTM/blob/main/scripts/downstream_02_molecule_edit_step_02_MoleculeSTM_Latent_Optimization.py), different $\lambda$ are sumed up by 'any' operation. Thus, changable hyperparamers are only the learning rate. But you only give 2 choice, I wonder how to get such results with 2 accuracy number.
![Supplementary Information](https://github.com/user-attachments/assets/346aaf71-5359-4b83-b67d-07e6ef686c1a)
![Table 13](https://github.com/user-attachments/assets/8aadd44a-affa-4649-920f-ab0b57c44872)

**Comment by chao1224** [comment]
Hi @spoilvoid , we were using seeds 1,2,3.
